### ISSUE TITLE
[Global Styles] Red alert button's colors are not accessible

### ISSUE THREAD

**Issue by andycochran** [zurb/foundation-sites#7377]
**How can we reproduce this bug?**
Go to Global Styles Page
Use WebAIM Color Contrast Checker (http://webaim.org/resources/contrastchecker)
Enter "FFFFFF" as "foreground" color.
Enter "ec5840" as "background" color.

**What did you expect to happen?**
Contrast ratio for normal text should be 4.5:1 or higher
WCAG AA criteria should pass

**What happened instead?**
Contrast ratio is below standard.
See: http://webaim.org/resources/contrastchecker/?fcolor=ffffff&bcolor=ec5840

**Comment by andycochran** [comment]
The slightest change that allows AA to pass is #cc4b37 (keeping the same red hue and saturation, changing only the brightness).

**Comment by gakimball** [comment]
I'm collecting all of the contrast issues into one: #7387
